pages/linux/beep.md

```diff
@@ -0,0 +1,11 @@
+# Beep
+
+> A utility to beep the pc speaker.
+
+- Send a series of beeps to the speaker at a specified freqency and duration:
+
+`beep -f {{frequency}} -l {{duration}} -r {{repetitions}}`
```

Comment:
Is there a simpler example that could be used for the first one? Say, with one or two options at most.